### ISSUE TITLE
Cannot access Subscription Management page from FxA Settings

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#10342]
Prerequisites:
1. Have a FxA account with an active subscription
2. Be signed in

Steps to reproduce:
1. Navigate to FxA settings page (accounts.firefox.com)
2. Click "Paid Subscriptions" in the left sidebar.

Expected:

* Taken to subscriptions.firefox.com

Actual:

* Presented with a login screen with a field to enter a password
* When the correct password (and optionally, verification code) are entered; redirected back to accounts.firefox.com.

Additional notes:

* Firefox release 91.0.2 on Mac OS Big Sur 11.5.2
* Barry could not reproduce on prod in Firefox release 91.0.2 on Windows. Zach and Barry both have 2FA setup, but unlike Zach, Barry did not have a secondary email address set up.
* Zach was able to access Subscription Management on his phone after entering in his credentials.
* Phone: IPhone 12 Pro, Firefox iOS app

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXA-3947)

**Comment by data-sync-user** [comment]
➤ Bianca Danforth commented:

Zachery Sutton , can you try this with a different Firefox profile on the same computer (see this SUMO article for how to create a new profile ( https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles?redirectslug=profile-manager-create-and-remove-firefox-profiles&redirectlocale=en-US )) or on another computer?

**Comment by data-sync-user** [comment]
➤ Zachery Sutton commented:

Hi Bianca Danforth , You must be a genius! I created a new profile and it worked! Do I need to delete my other profiles? Here is a short video.

https://cln.sh/JZU4HZ ( https://cln.sh/JZU4HZ|smart-link ) 

https://cln.sh/HWbiZY ( https://cln.sh/HWbiZY|smart-link ) 

Thanks

Zachery

**Comment by data-sync-user** [comment]
➤ Zachery Sutton commented:

Everything is working fine now. I changed my profile and everything is working!

**Comment by data-sync-user** [comment]
➤ Bianca Danforth commented:

Okay great thanks for confirming. Yeah Firefox Profiles store a lot of user-specific information ( https://support.mozilla.org/en-US/kb/profiles-where-firefox-stores-user-data#w_what-information-is-stored-in-my-profile ). It’s hard to say for sure what it could have been, but most likely somehow the cookies or client-side storage that FxA uses got corrupted.

FWIW I also could not reproduce the issue in prod (Firefox Nightly 93.0a1 on MacOS 11.5) with 2FA and a secondary email set up. I am going to cancel it for now.

**Comment by data-sync-user** [comment]
➤ Bianca Danforth commented:

Zachery Sutton By chance, do you still have your old profile? Can you check to see if the email address on the login screen after clicking “Paid Subscriptions” is the same email address that you signed into FxA Settings (accounts.firefox.com ( http://accounts.firefox.com )) with? Since you mentioned you had a secondary email address set up…

**Comment by data-sync-user** [comment]
➤ Zachery Sutton commented:

Hi Bianca Danforth,

I still have it. I just checked it. Yes, after I click on paid subscriptions, the email is the same as the email I signed into FxA with ( zacherysutton@gmailcom) last week. It was switching between my primary and secondary emails. I guess that is where the issue was happening. Now I am not having any issues with my old profile.

Quick Video Here ( https://cln.sh/buPST6 )

Thanks

Zachery
